### DISCUSSION TITLE
Help offered and NMEA2000

### DISCUSSION THREAD

**Discussion by hugorobi** [peterantypas/maiana#22] [Ideas]
Hi,

I you want beta tester in north america (Canada), feel free to contact me !

I have basic skill in electronic but advanced skill in programming and testing.

I can also help on V2 (or 1.1) of the transponder to make it NMEA2000 compatible with the excellent work of Timo Lappalainen (https://github.com/ttlappalainen/NMEA2000), if he agree.

Nice job !

Hugo

**Comment by peterantypas** [comment]
Thanks. Will definitely keep this in mind.
